mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -282,13 +282,15 @@ emulatedVectorLoad(OpBuilder &rewriter, Location loc, Value base,
                    OpFoldResult linearizedIndices,
                    int64_t numEmultedElementsToLoad, Type origElemType,
                    Type emulatedElemType) {
-  auto scale = emulatedElemType.getIntOrFloatBitWidth() /
-               origElemType.getIntOrFloatBitWidth();
+  auto elementsPerContainerType = emulatedElemType.getIntOrFloatBitWidth() /
```

Comment:
`elemPerContainerType` to make it a little bit shorter?

Reply:
Thanks for the suggestion! I'd like to introduce something even less ambiguous, so let me propose:
* `emulatedPerContainerElem`.

I'll send an update shortly and you can tell me what you think :)